### PR TITLE
Add files for dealing with empanelment.

### PR DESCRIPTION
**Pull request by mhoover-gallup**
This PR adds two R scripts, one that delineates input lists into divided, per-week empanelment lists for use with Qualtrics. The other touches up data coming out of Qualtrics and empanelment to provide data for Volunteer Science to load for game play. 

Note, both could use touch-up as they are not totally push-button functional.

### CODE REVIEW

**Comment by mhoover-gallup** [comment]
@pablo-diego-rosell this needs review ASAP.

File: cycle2/empanelment/create_empanelment_lists.r [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# ngs2 cycle 2 empanelment lists
+# for questions, contact: matt hoover (matt_hoover@gallup.com)
+
+# clean environment and install packages that are required
+rm(list = ls())
+local <- TRUE # change to FALSE if getting data from the dod_clients drive
+
+# determine system environment and define constants for use
+if(Sys.info()['sysname'] == "Windows") {
+    dd <- "W:/DARPA_NGS2/CONSULTING/Analytics/cycle2/data"
```
Comment (pablo-diego-rosell) [inline cycle2/empanelment/create_empanelment_lists.r]: Replacing "W:/" with "//gallup/dod_clients" below works better for me:

dd <- "//gallup/dod_clients/DARPA_NGS2/CONSULTING/Analytics/cycle2/data"
od <- "//gallup/dod_clients/DARPA_NGS2/CONSULTING/Analytics/cycle2/output"

File: cycle2/empanelment/cycle2_empanelment_vs_prep.r [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+# load data
+c <- read.csv('cycle2/data/cycle2_empanelment_24oct2018.csv',
+              sep = ',', header = TRUE, stringsAsFactors = FALSE)
+c <- c[c$Status == 0 & c$Q3 == 1, ]
+
+# impute missing data for scale values
+c$nid <- 1:nrow(c)
+hd_values <- hot.deck(c[, grep(imputation_variables, names(c))], method = 'best.cell',
+                      cutoff = 1, sdCutoff = 3)
```
Comment (pablo-diego-rosell) [inline cycle2/empanelment/cycle2_empanelment_vs_prep.r]: Please comment on the rationale for "sdCutoff = 3"

File: cycle2/empanelment/cycle2_empanelment_vs_prep.r [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+trans_leadership <- function(x, choice=c('sum', 'mean')) {
+    if(choice == 'sum') {
+        tmp <- apply(x[, grep('Q107_', names(x))], 1, sum)
+    } else {
+        tmp <- apply(x[, grep('Q107_', names(x))], 1, mean)
+    }
+    return(tmp)
+}
+
+# instructions
```
Comment (pablo-diego-rosell) [inline cycle2/empanelment/cycle2_empanelment_vs_prep.r]: I would move these instructions to the top, I was confused for example by the missing 'Q107_7' variable until I read this comment.

**Comment by mhoover-gallup** [comment]
Closing in favor of #6. Will port comments over.

**Pull request closed without merging**